Assets/Mirror/Runtime/CustomAttributes.cs

```diff
@@ -10,51 +10,79 @@ public class NetworkSettingsAttribute : Attribute
         public float sendInterval = 0.1f;
     }
 
+    /// <summary>
+    /// Automatically syncs a value from the server to clients.
+    /// </summary>
     [AttributeUsage(AttributeTargets.Field)]
     public class SyncVarAttribute : Attribute
     {
+        ///<summary>A function that should be called when the value changes on the client.</summary>
```

Comment:
Keep consistent format please,   the other ones have a <summary>  in a separate line